ios/Sources/HomeFeature/HomeContentFeature.swift

```diff
@@ -0,0 +1,70 @@
+import ComposableArchitecture
+import Model
+import Repository
+
+public struct HomeContentState: Equatable {
```

Comment:
The word `content` is little tricky. Maybe `list` is little better than `content`